src/activities/components/ActivitySeriesEdit.vue

```diff
@@ -295,6 +280,11 @@
       </div>
 
       <div class="row justify-end q-gutter-sm q-mt-lg">
+        <QToggle
+          v-model="showPreview"
+          label="Show preview"
```

Comment:
i18n

Reply:
switched to just "Preview" as there was some existing translation...